andela/wg-justice-league
# #154240259 Update muscle list

**What this PR do?**

- Update muscle list

**Description of the task to be completed**

- The list of available muscles as well as the exercises could be updated
    New muscles:
    lateral deltoid - 17
    Posterious Deltoid - 19
    Forearms - 20

**How can this be manually tested?
 
- After running the command **invoke load-fixtures --settings-path=settings.py** at the console and navigating to the muscle overview, we see that the muscle list is updated.

- Before updating muscle list
<img width="1268" alt="screen shot 2018-02-06 at 4 38 13 pm" src="https://user-images.githubusercontent.com/6010217/35958893-2e687f9e-0cb4-11e8-9f30-5ab2620b51df.png">


<img width="1428" alt="screen shot 2018-02-06 at 4 38 33 pm" src="https://user-images.githubusercontent.com/6010217/35958928-5136051e-0cb4-11e8-992c-a20ed5697c82.png">

- After updating muscle list

<img width="1133" alt="screen shot 2018-02-08 at 9 47 23 am" src="https://user-images.githubusercontent.com/6010217/35959103-235b53e6-0cb5-11e8-9474-31d57d5234cd.png">

<img width="1131" alt="screen shot 2018-02-08 at 9 48 17 am" src="https://user-images.githubusercontent.com/6010217/35959121-3dcbe95c-0cb5-11e8-9351-d01d8e7103ac.png">

<img width="1031" alt="screen shot 2018-02-08 at 9 49 04 am" src="https://user-images.githubusercontent.com/6010217/35959141-5b8f8f66-0cb5-11e8-8139-dcd1a8b4bb95.png">

**Relevant pivotal tracker stories**

#154240259
